clang/www/analyzer/index.html

```diff
@@ -121,10 +80,25 @@ <h3 style="margin:0px;padding:0px">Mac OS X</h3>
  <b class="spiffy5"></b></b>
  <div class="spiffyfg">
   <div style="padding:15px">
-   <h3 style="margin:0px;padding:0px">Other Platforms</h3>
-   <p>For other platforms, please follow the instructions for <a
-   href="/installation#OtherPlatforms">building the analyzer</a> from
-   source code.<p>
+    <p>The static analyzer is part of Clang. Please refer to the
+    <a href="https://releases.llvm.org/download.html">LLVM
+    releases page</a> for further instructions.</p>
```

Comment:
```suggestion
    download page</a> for further instructions.</p>
```

It calls itself the "Download page".

Reply:
True, I had the GitHub releases page in mind when I typed this, perhaps.